### PR TITLE
Improve changelog

### PR DESCRIPTION
**Pull request by vbauzysvmware**
Went through commits and updated changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
 ## 2.1.0 (March 25, 2019)
 
-ARCHITECTURAL:
+BREAKING CHANGES:
```
Comment (Didainius) [inline CHANGELOG.md]:
```suggestion
NOTES:
```

I think these are rather NOTES than BREAKING CHANGES. User experience is not broken and as per Hashicorp `This should only be present in major version upgrades.`

Comment (vbauzysvmware) [reply CHANGELOG.md]: ok. :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
+* **New Resource:** vApp network - `vcd_vapp_network`
+* resource/vcd_vapp_vm has ability to use vApp network 
 
 IMPROVEMENTS:
 
-* Insert Media now supports force ejecting on running VM
+* resource/vcd_inserted_media now supports force ejecting on running VM
+
+BUG FIXES:
+* [68] Issue - added vAPP status handling when env is very fast
```
Comment (dn1s) [inline CHANGELOG.md]: Maybe linking the issue would make more sense.

Comment (dn1s) [reply CHANGELOG.md]: Like this:
```
[[68](https://github.com/terraform-providers/terraform-provider-vcd/issues/68)]
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,19 +1,25 @@
-## 2.1.0 (February 5, 2019)
+## 2.1.0 (March 25, 2019)
```
Comment (lvirbalas) [inline CHANGELOG.md]: 👍 for the optimism!

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,19 +1,25 @@
...
 * Project switched to using Go modules, while `vendor` is left for backwards build compatibility only. It is worth having a
 look at [README.md](README.md) to understand how Go modules impact build and development.
 * Project dependency of github.com/hashicorp/terraform updated from v0.10.0 to v0.11.13.
 
 FEATURES:
 
-* New Independent disk resource - `vcd_independent_disk`
-* Ability to attach independent disk in `vcd_vapp_vm` 
+* **New Resource:** disk resource - `vcd_independent_disk`
+* resource/vcd_vapp_vm ahas ability to attach independent disk in
```
Comment (lvirbalas) [inline CHANGELOG.md]: A typo and unfinished sentence?

File: CHANGELOG.md (hunk 2 above)
Comment (lvirbalas) [reply CHANGELOG.md]: Links would be nice as we already have some in 2.0.0 log below. Looking at the Terraform [changelog guide](https://www.terraform.io/docs/extend/best-practices/versioning.html#changelog-specification), they suggest adding the link at the end of sentence, e.g.:
`resource/subnet: Now better [GH-22, GH-32]`

File: CHANGELOG.md (hunk 3 above)
Comment (Didainius) [reply CHANGELOG.md]: I suggest - as per  https://www.terraform.io/docs/extend/best-practices/versioning.html#changelog-specification
> NOTE: For HashiCorp released providers, the release process will replace the "Unreleased" header with the current date. This line must be present with the target release version to successfully release that version.

```suggestion
## 2.1.0 (Unreleased)
```

File: CHANGELOG.md (hunk 2 above)
Comment (vbauzysvmware) [reply CHANGELOG.md]: We have to follow Harshicorp Guidelines. So we are left with using [GH-**]

File: CHANGELOG.md (hunk 3 above)
Comment (vbauzysvmware) [reply CHANGELOG.md]: Ok. For now Unreleased :)

File: CHANGELOG.md (hunk 4 above)
Comment (vbauzysvmware) [reply CHANGELOG.md]: Fixed

**Comment by vbauzysvmware** [comment]
Latest changes pushed

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,19 +1,27 @@
...
+* **New Resource:** vApp network - `vcd_vapp_network` [GH-155]
+* resource/vcd_vapp_vm has ability to use vApp network [GH-155]
 
 IMPROVEMENTS:
 
-* Insert Media now supports force ejecting on running VM
+* resource/vcd_inserted_media now supports force ejecting on running VM [GH-184]
+
+BUG FIXES:
+* add vAPP status handling when environment is very fast [GH-68]
```
Comment (lvirbalas) [inline CHANGELOG.md]: vAPP -> vApp

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,19 +1,27 @@
...
+* add additional validation to check if vApp template is ok [GH-157]
```
Comment (lvirbalas) [inline CHANGELOG.md]: As we started using "resource/..." pattern for the change log, let's add that to the above two lines as well to be consistent.

File: CHANGELOG.md (hunk 5 above)
Comment (vbauzysvmware) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 6 above)
Comment (vbauzysvmware) [reply CHANGELOG.md]: Added, though not sure is good way, as one of fix in govcd and just inherited.

**Pull request merged**